content/docs/start/model-management/model-cicd.md

```diff
@@ -0,0 +1,234 @@
+
+## Connecting model registry actions to your CICD
+
+As we [noted](/docs/start/model-management/model-registry#GTO-tip) in the model
+registry chapter, all DVC model registry actions are captured in your Git
+repository as Git tags with a specific format.
+
+This also means that we can create CICD actions in our Git repository which will
+be triggered whenever versions are registered or stages are assigned.
+
+In the following, we will have a look at an example CICD workflow on GitHub
+which runs whenever we assign a version of our model to the "prod" stage in the
+model registry. The workflow simulates model deployment without the need to
+actually set up a deployment environment (so that you can test it easier) but it
+does include all the ingredients needed in an actual deployment job or any other
+CICD action.
+
+<admon type="tip">
+
+To see a real-world model deployment example you can check out a
+[similar workflow in our example repository](https://github.com/iterative/example-get-started-experiments/blob/main/.github/workflows/deploy-model-sagemaker.yml)
+which deploys a specific version of the model to an Amazon Sagemaker endpoint
+for inference whenever it is assigned to a stage.
+
+</admon>
+
+Go to the `.github/workflows/deploy-model-template.yml`. This is the file that
+GitHub uses to run our CICD workflow. You can see
+[runs of this workflow](https://github.com/iterative/example-get-started-experiments/actions/workflows/deploy-model-template.yml)
+in our example repository.
+
+At the beginning of the workflow file you will see this code
+
+```yaml
+on:
+  # the workflow is triggered whenever a tag is pushed to the repository
+  push:
+    tags:
+      - '*'
+```
+
+The code tells GitHub to run the workflow every time a tag is pushed to the
+repository.
+
+This means that the workflow will run whenever we run model registry actions,
+but we also want it to limit to specific ones for our specific workflow. That's
+where our GTO GitHub action comes into play - in the "parse" job of our workflow
+it parses all tags and if they are GTO tags, it gives us the name of the model,
+its version, stage (if any) and the event in the model registry.
+
+This is captured in the "parse" job which you can simply copy and paste into
+most CICD jobs of your own.
+
+```yaml
+# This job parses the git tag with the GTO GitHub Action to identify model registry actions
+parse:
+  runs-on: ubuntu-latest
+  steps:
+    - uses: actions/checkout@v3
+    - name: 'Parse GTO tag'
+      id: gto
+      uses: iterative/gto-action@v2
+  outputs:
+    event: ${{ steps.gto.outputs.event }}
+    name: ${{ steps.gto.outputs.name }}
+    stage: ${{ steps.gto.outputs.stage }}
+    version: ${{ steps.gto.outputs.version }}
+```
+
+<admon type="tip">
+
+If you cannot or don't want to use the GTO GitHub Action you can also use GTO
+directly with the [gto check-ref](/doc/gto/command-reference/check-ref) command.
```

Comment:
Not a blocker, but it might be good to be more clear here. The GTO Github Action only works on GitHub. If you aren't using GitHub, you can use GTO directly.

Reply:
```suggestion
If you are not using GitHub or if you don't want to use the GTO GitHub Action you can also use GTO
directly with the [gto check-ref](/doc/gto/command-reference/check-ref) command.
```